package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "grunt-wordpress-builder",
+  "private": false,
+  "devDependencies": {
+    "grunt": "^0.4.5",
+    "grunt-autoprefixer": "^3.0.3",
+    "grunt-browser-sync": "^2.1.3",
+    "grunt-contrib-sass": "^0.9.2",
+    "grunt-contrib-watch": "^0.6.1",
+    "grunt-notify": "^0.4.1",
+    "load-grunt-tasks": "^3.3.0",
+    "node-sass": "^3.4.2"
+  },
+  "engines": {
+    "node": ">=5.0"
+  },
+  "description": "Comics!",
+  "version": "1.0.0",
+  "main": "",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/asanchez78/comicdb"
+  },
+  "keywords": [
+    "sass",
+    "coffeescript",
+    "typescript",
+    "grunt",
+    "autoprefixer",
+    "wordpress",
+    "json"
+  ],
+  "author": "Anthony Sanchez and Sean Loyless",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/asanchez78/comicdb/issues"
+  },
+  "homepage": "https://github.com/asanchez78/comicdb"
+}
```

Comment:
This is the Node.JS Config file. When you run `npm install` in terminal, this is the file it reads to know what packages to install.